setup.py

```diff
@@ -57,8 +57,8 @@ def _dbt_sqlserver_version():
         ]
     },
     install_requires=[
-        'dbt-core~=0.19.0',
+        "dbt-core>=0.19.0",
```

Comment:
I'm fairly certain that we want to keep the tilde character bc (according to this [SO answer](https://stackoverflow.com/a/39590286/3842610)), it is a combination of to a combination of the following clauses:
- `>= 0.19.0` and
- `== 0.19.*`

This way a user will get an error if they try to install `dbt-core==0.20.0` before we cut a compatible release.

Reply:
Sure, that is true. Though, we might miss when there is a new minor version and thus forget to leverage that.

The projects I use as examples - like `[tox](https://github.com/tox-dev/tox/blob/master/setup.cfg)` or `[pre-commit](https://github.com/pre-commit/pre-commit/blob/master/setup.cfg)` - have lower bounded requirement version. Sometimes you see an upper bound for the major release `dbt-core>=0.19.0,<1.0.0`. 

I think the idea is that `dbt-core` should be backwards compatible for minor releases, however, it could of course happen that that is not the case.

Reply:
good point. no strong opinion here. @mikaelene might have one though!